### ISSUE TITLE
Problem in running setup_windows_cuda.py

### ISSUE THREAD

**Issue by debbie-ly77** [msracver/Deformable-ConvNets#107]
Hi everyone, I'm new in Mxnet. When I'm trying to run the demos, I faced the following problem. 

The platform I use:
windows10+python2.7.14+CUDA8.0
anaconda
mingw4.8.0

When I run the init.bat, there is an error in "python setup_windows_cuda.py build_ext --inplace" of folder "nms". The error information is:

c:/mingw/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/4.8.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lcublas
collect2.exe: error: ld returned 1 exit status
error: command 'C:\\MinGW\\mingw64\\bin\\g++.exe' failed with exit status 1

I suppose it is because of the mingw and I tried both 4.7.0 and 4.8.0. Both of them are useless in solving this problem. 

Do you have any suggestion? I will appreciate any kind of help.

**Comment by YuwenXiong** [comment]
You need to install visual studio, nvcc only support visual studio on Windows.

**Comment by debbie-ly77** [comment]
Yes. Actually I have vs2015 and vs2013 on my machine, but I still don't know why they don't work. Do I need to add some specific path on the environment?

**Comment by YuwenXiong** [comment]
You need to use vs 2015 rather than mingw to compile.
